pkg/ddc/factory.go

```diff
@@ -34,13 +35,14 @@ var buildFuncMap map[string]buildFunc
 
 func init() {
 	buildFuncMap = map[string]buildFunc{
-		"alluxio":  alluxio.Build,
-		"jindo":    jindo.Build,
-		"jindofsx": jindofsx.Build,
-		"goosefs":  goosefs.Build,
-		"juicefs":  juicefs.Build,
-		"thin":     thin.Build,
-		"efc":      efc.Build,
+		"alluxio":    alluxio.Build,
+		"jindo":      jindo.Build,
+		"jindofsx":   jindofsx.Build,
```

Comment:
How about puting jindo,jindofsx and jindocache together?

Reply:
done